cmds/portmaster-start/run.go

```diff
@@ -121,6 +121,20 @@ func getExecArgs(opts *Options, cmdArgs []string) []string {
 	if stdinSignals {
 		args = append(args, "--input-signals")
 	}
+
+	// see https://www.freedesktop.org/software/systemd/man/pam_systemd.html#type=
+	if xdgSessionType := os.Getenv("XDG_SESSION_TYPE"); xdgSessionType == "wayland" && opts.Identifier == "app/portmaster-app.zip" {
```

Comment:
Not sure about the impact of `os.Getenv`, but we could pull the `opts.Identifier == "app/portmaster-app.zip"` check outside and only check the ENV if we are actually trying to launch the app.

Reply:
environment variables are already part of the process memory and it's basically just a map lookup under a RLock/using a syscall so this check is negligible.